src/styles/app.css

```diff
@@ -1,5 +1,599 @@
+}
+/* vietnamese */
+@font-face {
+    font-family: "Open Sans";
+    font-style: normal;
+    font-weight: 800;
+    font-stretch: 100%;
+    font-display: swap;
+    src: url(../../public/fonts/Open_Sans/static/OpenSans-ExtraBold.ttf);
+    unicode-range: U+0102-0103, U+0110-0111, U+0128-0129, U+0168-0169,
+        U+01A0-01A1, U+01AF-01B0, U+1EA0-1EF9, U+20AB;
+}
+/* latin-ext */
+@font-face {
+    font-family: "Open Sans";
+    font-style: normal;
+    font-weight: 800;
+    font-stretch: 100%;
+    font-display: swap;
+    src: url(../../public/fonts/Open_Sans/static/OpenSans-ExtraBold.ttf);
+    unicode-range: U+0100-024F, U+0259, U+1E00-1EFF, U+2020, U+20A0-20AB,
+        U+20AD-20CF, U+2113, U+2C60-2C7F, U+A720-A7FF;
+}
+/* latin */
+@font-face {
+    font-family: "Open Sans";
+    font-style: normal;
+    font-weight: 800;
+    font-stretch: 100%;
+    font-display: swap;
+    src: url(../../public/fonts/Open_Sans/static/OpenSans-ExtraBold.ttf);
+    unicode-range: U+0000-00FF, U+0131, U+0152-0153, U+02BB-02BC, U+02C6, U+02DA,
+        U+02DC, U+2000-206F, U+2074, U+20AC, U+2122, U+2191, U+2193, U+2212,
+        U+2215, U+FEFF, U+FFFD;
+}
+
+/*Notícia Text 400*/
+/* vietnamese */
+@font-face {
+    font-family: "Noticia Text";
+    font-style: normal;
+    font-weight: 400;
+    font-display: swap;
+    src: url(../../public/fonts/Noticia_Text/NoticiaText-Regular.ttf);
+    unicode-range: U+0102-0103, U+0110-0111, U+0128-0129, U+0168-0169,
+        U+01A0-01A1, U+01AF-01B0, U+1EA0-1EF9, U+20AB;
+}
+/* latin-ext */
+@font-face {
+    font-family: "Noticia Text";
+    font-style: normal;
+    font-weight: 400;
+    font-display: swap;
+    src: url(../../public/fonts/Noticia_Text/NoticiaText-Regular.ttf);
+    unicode-range: U+0100-024F, U+0259, U+1E00-1EFF, U+2020, U+20A0-20AB,
+        U+20AD-20CF, U+2113, U+2C60-2C7F, U+A720-A7FF;
+}
+/* latin */
+@font-face {
+    font-family: "Noticia Text";
+    font-style: normal;
+    font-weight: 400;
+    font-display: swap;
+    src: url(../../public/fonts/Noticia_Text/NoticiaText-Regular.ttf);
+    unicode-range: U+0000-00FF, U+0131, U+0152-0153, U+02BB-02BC, U+02C6, U+02DA,
+        U+02DC, U+2000-206F, U+2074, U+20AC, U+2122, U+2191, U+2193, U+2212,
+        U+2215, U+FEFF, U+FFFD;
+}
+/* vietnamese */
+@font-face {
+    font-family: "Noticia Text";
+    font-style: normal;
+    font-weight: 700;
+    font-display: swap;
+    src: url(../../public/fonts/Noticia_Text/NoticiaText-Bold.ttf);
+    unicode-range: U+0102-0103, U+0110-0111, U+0128-0129, U+0168-0169,
+        U+01A0-01A1, U+01AF-01B0, U+1EA0-1EF9, U+20AB;
+}
+/* latin-ext */
+@font-face {
+    font-family: "Noticia Text";
+    font-style: normal;
+    font-weight: 700;
+    font-display: swap;
+    src: url(../../public/fonts/Noticia_Text/NoticiaText-Bold.ttf);
+    unicode-range: U+0100-024F, U+0259, U+1E00-1EFF, U+2020, U+20A0-20AB,
+        U+20AD-20CF, U+2113, U+2C60-2C7F, U+A720-A7FF;
+}
+/* latin */
+@font-face {
```

Comment:
I think you can keep only `latin` variations how I said in the message.

Reply:
And read this article
https://css-tricks.com/understanding-web-fonts-getting/#:~:text=TrueType%20(TTF)%3A%20TTF%20is,mobile%2C%20if%20you%20need%20it.

I think you should use WOFF fonts instead of TTF, but it's preferable that you serve all and the browser selects the best fit.

Reply:
```css
@font-face {
  font-family: 'MyWebFont';
  src: url('myfont.woff2') format('woff2'),
       url('myfont.woff') format('woff'),
       url('myfont.ttf') format('truetype');
}
```